plasmapy/diagnostics/charged_particle_radiography.py

```diff
@@ -1419,3 +1422,249 @@ def synthetic_radiograph(
             intensity = -np.log10(intensity / I0)
 
     return h * u.m, v * u.m, intensity
+
+
+# *************************************************************************
+# Film Stack & Layers
+# *************************************************************************
```

Comment:
Yeah, I'm good with that!

Reply:
Done. I didn't make it private because this isn't actually imported by the charged_particle_radiography file (yet) but is a separate and related tool.

Reply:
So, I moved this over but since `_detector_stacks` is private I would still like users to import from the `charged_particle_radiography` module, and also for references like 
`~plasmapy.diagnostics.charged_particle_radiography.Layer` to work. What's the best way to do that?